tests/Browser/UserSearchTest.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace Tests\Browser;
+
+use Tests\DuskTestCase;
+use Laravel\Dusk\Browser;
+use Tests\Browser\Pages\UserSearchPage;
+use Tests\Browser\Pages\HomePage;
+
+class UserSearchTest extends DuskTestCase
+{
+    /**
+     * Test visiting the User Search Page.
+     * Searching for a user by name (instead of
+     * valid email address) returns "No user found!" message.
+     */
+    public function testIncorrectUserSearchPage()
+    {
+        // $this->browse(function (Browser $browser) {
+        //     $browser->login()
+        //         ->visit(new UserSearchPage)
+        //         ->assertSeeIn('@title', 'Users');
+                // ->type('@search', 'taylor')
+                // ->press('Submit')
+                // ->assertSeeIn('@messages', 'No user found!');
+        // });
+
+        $this->browse(function (Browser $browser) {
+            $browser->visit(new HomePage)
+                    ->click('@login-button')
+                    ->assertPathIs('/register')
+                    ->clickLink('Log In')
+                    ->type('username', 'clee@dosomething.org')
+                    ->type('password', env('NORTHSTAR_PASSWORD'))
+                    ->press('Log In')
+                    ->assertPathIs('/campaigns');
```

Comment:
@DFurnes the redirect path is `/campaigns` and I can successfully get to that page! However, whenever I try to do anything next, it doesn't register. Will this be fixed by registering this as a `redirect_uri` in Northstar QA like you mentioned before?

Reply:
@DFurnes this was the last q - not sure if you saw! :)

Reply:
oh I just saw that I am able to add that as a `redirect_uri` actually on Aurora! But not sure if that fixes the problem so maybe I'm misunderstanding?

Reply:
Ah sorry, totally missed this! Yeah, you should be able to paste in the full URL for this into the client's "edit" form on Aurora. I can swing by if you want to walk through it IRL!

Reply:
@DFurnes if you wouldn't mind swinging by, that would be so helpful! I added it but am getting something returned that I'm not expecting (confirms I'm on the `/campaigns` page but when I dump it out, it is a lot of code with `new TypeError("cannot call a class as a function"` etc.)

Reply:
Ooooh weird! Let's definitely dig into that tomorrow.